src/Orleans/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Newtonsoft.Json": "7.0.1",
+    "Newtonsoft.Json": "9.0.1",
```

Comment:
It's not Obsolete, is it? If it is, then we must update. Otherwise we should not do this breaking change, as it would force users to have to update even if they are not ready for it yet. We use the minimum version required by the functionality that we need, as long as it is forward compatible and it works even if users want to be on the latest (with binding redirects).
In tests it's fine to use whatever version we want, as it does not force anything on end users.